nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenStat.scala

```diff
@@ -82,6 +82,7 @@ trait NirGenStat { self: NirGenPhase =>
       val body   = cd.impl.body
 
       buf += Defn.Class(attrs, name, None, Seq.empty)
+      genReflectiveInstantiation(cd)
```

Comment:
I don't think we should generate reflective instantiation info for structs. Or at least we should very carefully consider the consequences of doing so.

FTR, no reflective instantiation info are ever generated for JS classes in Scala.js, so there is precedent for ignoring that for interop-specific "stuff".